src/unity/schemas/IdentityAccount.yml

```diff
@@ -0,0 +1,18 @@
+properties:
+  id:
+    type: number
+    description: account id in quartz
+  name:
+    type: string
+    description: name of the account
+  type:
+    $ref: './AccountType.yml'
+  paygCreditStartDate:
+    type: string
+    nullable: true
+    description: a timestamp indicates if an account is part of the payg credit experiment, nil if not
+required:
+  - id
+  - name
+  - type
+  - paygCreditStartDate
```

Comment:
If `nil` is possible for this field, then maybe we don't need to mark it as `required` so it can be safely omitted?